app/styles/_base.scss

```diff
@@ -109,6 +109,21 @@ input {
   font-weight: inherit;
 }
 
+button.link {
```

Comment:
I think we can remove border, letter-spacing, and width here as well.

Reply:
Actually you should update `width` to `auto`, the button will size to fit the text.

Reply:
About letter-spacing, all buttons use 0.2px by default, which visibly is a little different from regular text or links in FxA. Shouldn't we keep the text in a .link button look similar to that of a regular link?

Reply:
Sure, you can keep letter spacing in that case.